RecaptchaValidator/RecaptchaValidator.js

```diff
@@ -0,0 +1,88 @@
+const superagent = require('superagent');
```

Comment:
nit: note that we depend on `node-fetch` already in `near-api-js`, but also should be ok to use `superagent` if it's more convenient for the job (given it's not public js bundle)

Reply:
It would take a fair amount of code to re-create superagent's error handling and retry logic, so if this isn't a blocker I'll leave it using `superagent` here.

Also, for my understanding, I'm curious why you wouldn't consider using `superagent` for a public js bundle; is the concern around bundle size, or are there other reasons that would restrict us from using it there?